.github/workflows/CI.yml

```diff
@@ -52,6 +52,25 @@ jobs:
           files: ./coverage.xml
           flags: python-test
 
+  console-api-tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.python-version }}
+      - run: |
+          python -m pip install --upgrade pipenv
+          pipenv install --deploy --dev
+          pipenv run coverage run --source='.' manage.py test console_api
+          pipenv run coverage xml
+      - uses: codecov/codecov-action@v4
+        with:
+          files: ./coverage.xml
+
```

Comment:
Note for a follow up change in the sonarqube stuff to incorporate this

Reply:
Created a JIRA issue: [Add Console API test coverage to Sonarqube](https://opensearch.atlassian.net/browse/MIGRATIONS-1876)